src/Wrapper.js

```diff
@@ -311,6 +333,102 @@ class Wrapper extends React.PureComponent {
   }
 }
 
+const SwipeContainer = (() => {
+  let _progress = 0
```

Comment:
Changing the `state` renders a re-render, I want to keep progress without re-rendering.

Reply:
Sorry I meant the instance, not the state: `this._progress`

Reply:
Ahh, hmm, this is a personal preference: instead of trying to say this property is "private" I hide it using a closure.
Let me know if there is a team convention here.

Reply:
Not really a team convention, it’s just that it could theoretically lead to issues if we start using other instances of this component, since they would share a common variable.

Reply:
> it could theoretically lead to issues if we start using other instances of this component

If that were the case (more than one instance) I would've not used this approach.

I'm thinking I could add the swipe to close behavior to the `SignerPanel` so I'll set this as an instance prop to avoid forgetting about it.

Reply:
Do you think it would be much of a problem to just do `this._progress` instead of a shared `_progress`? Just thinking that it might be counter intuitive once we have it in a separate file, like any other component, except this one should only be declared once in an app, only because of this variable.

Reply:
>Do you think it would be much of a problem to just do `this._progress` instead of a shared `_progress`?

No problem at all, sorry if I did not communicate this clearly enough: I've already updated my lcoal code to use `this._progress`, will push in a while

Reply:
(actually `this._previousProgress`)

Reply:
Awesome 🤙